OracleWebLogic/samples/12213-webtier-apache/README.md

```diff
@@ -0,0 +1,95 @@
+Apache Web Server with Oracle WebLogic Server Proxy Plugin on Docker
```

Comment:
@mriccell doesn't it make more sense to call this the "WebLogic Server Proxy Plugin for Apache" image, i.e. make the WebLogic plugin the primary focus, not Apache?

You're the PM, so whatever you say goes, but it just seems weird to me to have all this Apache stuff when really it's about the WebLogic plugin bits. :)